frontend/src/metabase/components/BrowseApp.jsx

```diff
@@ -72,7 +72,12 @@ export class SchemaBrowser extends React.Component {
                 </Grid>
               </Box>
             ) : (
-              <TableBrowser {...this.props} />
+              <TableBrowser
+                {...this.props}
+                params={{ ...this.props.params, schemaName: schemas[0].name }}
+                // hide the schema since there's only one
```

Comment:
Is there anything preventing `schemas.length` from being 0?

Reply:
Ah yeah good call! I think that could happen if there were no tables in a db.